front/src/pods/trainer/trainer.container.tsx

```diff
@@ -32,7 +32,7 @@ export const TrainerContainer = () => {
 
     setSocket(localSocket);
 
-    localSocket.on(SocketOuputMessageLiteral.MESSAGE, msg => {
+    localSocket.on(SocketOuputMessageLiteral.MESSAGE, (msg) => {
```

Comment:
You should install prettier in your Visual Studio Code and enable it on save:
https://www.codereadability.com/automated-code-formatting-with-prettier/